src/screens/HelpScreen.tsx

```diff
@@ -1,11 +1,36 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { Component } from 'react';
+import { Image, StyleSheet, Text, View, Linking, Platform } from 'react-native';
 import baseStyle from '../styles/base';
+import PhoneButton from '../components/PhoneButton';
 
 export default function HelpScreen() {
+  function dialCall() {
+    let phoneNumber = '';
+
+    if (Platform.OS === 'android') {
+      phoneNumber = 'tel:${7344873387}';
+    } else {
+      phoneNumber = 'telprompt:${7344873387}';
+    }
+    Linking.openURL(phoneNumber);
+  }
+
   return (
     <View style={styles.container}>
-      <Text>Help Screen</Text>
+      <View style={styles.logoContainer}>
+        <Image
+          style={styles.logo}
+          source={require('../../assets/seeus-logo.png')}
+        />
+        <Text>Contact SEEUS</Text>
+        <PhoneButton
+          //  label="734.487.3387"
+          style={styles.loginButton}
+          labelStyle={styles.loginButtonText}
+          showShadow={true}
+          onPress={() => dialCall()}
```

Comment:
Once the handler code and styling is all in the PhoneButton component, I believe that none of these props will be needed.

Will be good to have as much code in the component as possible so it doesn't have to be duplicated in other screens where PhoneButton will be used.